### DISCUSSION TITLE
Open PDF to Right also opening system PDF viewer

### DISCUSSION THREAD

**Discussion by ricraftis** [RyotaUshio/obsidian-pdf-plus#156] [Q&A] [answered]
I am importing my articles from Zotero for annotation. I then open the PDF from the left hand file index in Obsidian. I have a symlink for all my Zotero PDF attachments. When I right click on the PDF for the article I am working on and "Open to the Right", it also opens the PDF in the system PDF viewer as well as a right hand pane in Obsidian. Although this can be useful for a zoomed perspective I would like to know if this feature can be switched on and off? 

Thanks

**Comment by RyotaUshio** [comment, marked as answer]
I think you've turned on this option:

![Untitled](https://github.com/RyotaUshio/obsidian-pdf-plus/assets/72342591/6e8c6d18-3f85-4400-98ea-b4160e95a550)

Disable it to prevent the system viewer from opening.

---

If this is not the case, I believe the behavior is caused by another plugin. There are some plugins that open PDFs in the system viewer.

**Comment by ricraftis** [comment]
That was the issue. Your plugin is so powerful and with great power comes steep learning curves. I am now ready to start doing article and video on using the plugin with Zotero article imports. It is an amazing tool for that use case. Many thanks.
